cmd/config-manager/main.go

```diff
@@ -43,7 +44,26 @@ var (
 	log = logrus.StandardLogger()
 )
 
+type nriConfig struct {
+	registrationTimeout string
+	requestTimeout      string
+}
+
 func main() {
+	var cfg nriConfig
+
+	flag.StringVar(&cfg.requestTimeout,
+		"nri-plugin-request-timeout", "", "configure NRI plugin request timeout")
+	flag.StringVar(&cfg.registrationTimeout,
+		"nri-plugin-registration-timeout", "", "configure NRI plugin registration timeout")
```

Comment:
instead of string, can we perhaps use `flag.Int()`? One obvious benefit is that we can check if the timeout value is valid.

Reply:
Again I wanted to use the exact same semantics as for the corresponding command line options in CRI-O. So these are actually `time.Duration`s.

Reply:
And we do check now if the timeout value is valid in `nriConfig.check()` using `time.ParseDuration()`.